js_modules/dagit/packages/core/src/nav/useRepositoryLocationReload.tsx

```diff
@@ -314,7 +315,6 @@ export const buildReloadFnForLocation = (location: string) => {
     >({
       mutation: RELOAD_REPOSITORY_LOCATION_MUTATION,
       variables: {location},
-      fetchPolicy: 'no-cache',
```

Comment:
couldn't figure out the motivation for this from the blame rev @bengotow

Reply:
This seems like a reasonable change to make now that there is a value to be cached, since before it was just `id`. But I also don't know/remember the motivation for `no-cache` here.

Reply:
Hey gang, I'm afraid I don't remember this change either. Based on my understanding of how this fetchPolicy stuff works, this was preventing apollo from putting the result data into the cache, but doing that should also be fine. 👍